client/src/components/questions.tsx

```diff
@@ -8,66 +8,75 @@ import React from "react";
 import { useSelector } from "react-redux";
 import { MuiThemeProvider, createMuiTheme } from "@material-ui/core/styles";
 import ScrollingQuestions from "components/scrolling-questions";
-import { State, MentorData } from "store/types";
+import { State } from "types";
+import withLocation from "wrap-with-location";
 
 const theme = createMuiTheme({
   palette: {
     primary: { main: "#1B6A9C" },
   },
-  // @ts-ignore
-  typography: { useNextVariants: true },
+  // eslint-disable-next-line
+  // typography: { useNextVariants: true },
```

Comment:
Can this be removed if not using typography style anymore?

Reply:
60b